### PR TITLE
Include the 'items' in the response when capturing orders

### PR DESCRIPTION
**Pull request by chrishulbert**
This was a requirement on a project I worked on, not sure if this will be helpful to the greater community? Thanks a lot for the great codebase. Cheers :)

### CODE REVIEW

File: order.go [hunk 1]
```diff
@@ -86,6 +86,7 @@ func (c *Client) CaptureOrder(orderID string, captureOrderRequest CaptureOrderRe
 	if err != nil {
 		return capture, err
 	}
+	req.Header.Set("Prefer", "return=representation") // Verbose response: https://developer.paypal.com/docs/api/orders/v2/#orders-authorize-header-parameters
```
Comment (roopakv) [inline order.go:89]: could we rather make this an option on the client?

as in after you do `NewClient` could we do

```
c.SetReturnRepresentation()
```

**Review by roopakv — COMMENTED** [review]
Thank you for doing this! just one suggestion

**Comment by chrishulbert** [comment]
Good idea, thanks, I will aim to implement something like that.

**Comment by plutov** [comment]
@chrishulbert any update on this PR?

**Comment by chrishulbert** [comment]
@plutov Apologies, but my current gig won't allow me the time to make the requested changes. I'll understand if you want to close this to tidy up.

**Comment by plutov** [comment]
@roopakv @chrishulbert I pushed a separate commit into main repository.

**Pull request closed without merging**